packages/core/src/common/k8s/resource-stack.ts

```diff
@@ -132,7 +132,9 @@ export class ResourceStack {
           labels["app.kubernetes.io/created-by"] = "resource-stack";
         }
 
-        resources.push(yaml.dump(entry));
+        resources.push(
+          yaml.dump(entry, { noArrayIndent: true, noCompatMode: true, noRefs: true, quotingType: '"', sortKeys: true }),
```

Comment:
Would it make sense to move this shared config to a central place or wrap this function so it's always consistent?